Utilities/bootstrap

```diff
@@ -518,6 +518,26 @@ def install_swiftpm(prefix, args):
     dest = os.path.join(prefix, "lib", "swift", "pm", "PluginAPI")
     install_dylib(args, "PackagePlugin", dest, ["PackagePlugin"])
 
+    # Install required resource bundles
+    resource_bundles = [
```

Comment:
No way we can keep this part in the swift-build repo? Will be easy to forget to update it...

Reply:
changed it to enumerate the build products for resource bundles instead of maintaining a list